packages/react/src/components/Storage/StorageManager/StorageManager/types.ts

```diff
@@ -0,0 +1,81 @@
+import { StorageAccessLevel } from '@aws-amplify/storage';
+import { StorageManagerDisplayText } from '../displayText';
+import { DropZoneProps, FilePickerProps } from '../DropZone/types';
+import { FileListProps } from '../FileList/types';
+import { ContainerProps } from '../FileListContainer/FileListContainer';
+import { FileListHeaderProps } from '../FileListHeader/FileListHeader';
+import { DefaultFile, StorageFile } from '../types';
+
+export type OnFilesChange = (
+  files: Array<Pick<StorageFile, 'file' | 'name' | 'status'>>
+) => void;
+
+export interface StorageManagerProps {
```

Comment:
is the StorageManager a 1:1 replacement of the FileUploader? Asking bc I don't see props like `variation` or `shouldAutoProceed` [or `shouldAutoUpload`] represented here.

Reply:
It's not 1:1, we've changed a lot of the APIs. Variation was removed because the button variation didn't work very well and didn't make sense. cc @dbanksdesign